### PR TITLE
Skarnet software: Rename attributes & split outputs

### PR DESCRIPTION
**Pull request by Profpatsch**
This (rather large) PR adds multiple outputs to all skarnet.org software. It also renames the attribute names of muilti-word packages to be more consistent.

In the future it might make sense to write a `buildSkarnetPackage` function to remove the duplication between most package derivations.

cc maintainer @pmahoney

### CODE REVIEW

**Comment by Profpatsch** [comment]
@GrahamcOfBorg build skalibs s6-linux-utils execline s6-portable-utils s6-dns s6-networking s6-rc s6

**Comment by FRidh** [comment]
@GrahamcOfBorg eval

File: nixos/doc/manual/release-notes/rl-1809.xml [hunk 1]
```diff
@@ -362,6 +362,15 @@ inherit (pkgs.nixos {
       The module <option>services.networking.hostapd</option> now uses WPA2 by default.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      <varname>s6Dns</varname>, <varname>s6Networking</varname>,
+      <varname>s6LinuxUtils</varname> and <varname>s6PortableUtils</varname>
+      renamed to
+      <varname>s6-dns</varname>, <varname>s6-networking</varname>,
+      <varname>s6-linux-utils</varname> and <varname>s6-portable-utils</varname> respectively.
```
Comment (Mic92) [inline nixos/doc/manual/release-notes/rl-1809.xml]: Should be `pkgs/top-level/aliases.nix` not more intuitive then reading release notes?

Comment (Profpatsch) [reply nixos/doc/manual/release-notes/rl-1809.xml]: Good idea.

**Comment by Profpatsch** [comment]
Huh,

```
clang-5.0: error: unable to execute command: posix_spawn failed: Resource temporarily unavailable
clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Makefile:136: background] Error 1
```

Since the other build on darwin was successful, I’ll assume this was just a temporary hickup of the build machine.

**Comment by Profpatsch** [comment]
I’d like to merge, any more comments? Vetoes?

**Pull request merged**